recipes/r-keyring/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-keyring
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/keyring_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/keyring/keyring_{{ version }}.tar.gz
+  sha256: f87f21c23fbd93dc18af68bfa0a8a7fcc4eea6412f9794681a7190be2069eec2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{native}}toolchain        # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}sed               # [win]
+    - {{posix}}grep              # [win]
+    - {{posix}}autoconf
+    - {{posix}}automake          # [not win]
+    - {{posix}}automake-wrapper  # [win]
+    - {{posix}}pkg-config
+    - {{posix}}make
+    - {{posix}}sed  # [win]
```

Comment:
sed and coreutils are duplicated

Reply:
I can merge directly after the fix, no need to wait for the CI

Reply:
Ah, I'm sorry I didn't notice that myself. I guess I should be on the lookout in future.